CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog for package code_coverage
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Keep *.info.cleaned tracefile
+
```

Comment:
please revert the changes to changelog - this gets auto generated when I do a release

Reply:
Done. Sry.